### PR TITLE
multicluster: Script to easily capture Envoy configs for multicluster development

### PR DESCRIPTION
**Pull request by draychev**
This PR adds a script, which attempts to automate the collection of __Envoy configuration__ during Multicluster development and demo/troubleshooting.

### CODE REVIEW

File: demo/get-multicluster-envoy-configs.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# This script automates the collection of Envoy config during Multicluster demo/development.
+# The configs are collected from Envoys in the ALPHA kube context (cluster).
+
+kubectx alpha > /dev/null
```
Comment (snehachhabria) [inline demo/get-multicluster-envoy-configs.sh:6]: lets add a not the the script stating that one needs `kubectx` to run this script

**Review by snehachhabria — APPROVED** [review]

File: demo/get-multicluster-envoy-configs.sh (hunk 1 above)
Comment (draychev) [reply demo/get-multicluster-envoy-configs.sh:6]: That's a good point.  I should use `kubectl` instead perhaps.

Comment (draychev) [reply demo/get-multicluster-envoy-configs.sh]: Switched to `kubectl config use-context alpha`

File: .gitignore [hunk 2]
```diff
@@ -76,3 +76,7 @@ docs/public/*
 node_modules/
 vendor/
 rootfs/
+
+
+# Envoy config captured during development
+envoy_*.json
```
Comment (allenlsy) [inline .gitignore:82]: There is a `Temp` folder in the repo. Not sure whether it's designed for scenarios like this. We can use put the debug files there, and add `Temp` in gitignore.

If it's not designed for this case, then we can use things like `tmp\`.

**Review by allenlsy — APPROVED** [review]

File: .gitignore (hunk 2 above)
Comment (draychev) [reply .gitignore:82]: Not a bad idea. I'll make a tweak to this script to use this in an upcoming PR.

**Pull request merged**